flink-connectors/flink-connector-files/src/main/java/org/apache/flink/connector/file/table/FileSystemConnectorOptions.java

```diff
@@ -156,6 +156,22 @@
                     .withDescription(
                             "The extractor class for implement PartitionTimeExtractor interface.");
 
+    public static final ConfigOption<String> PARTITION_TIME_EXTRACTOR_TIMESTAMP_FORMATTER =
+            key("partition.time-extractor.timestamp-formatter")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            Description.builder()
+                                    .text(
+                                            "It can combine with 'partition.time-extractor.timestamp-patter', "
```

Comment:
```suggestion
                                            "The formatter to format timestamp from string, it can be used with 'partition.time-extractor.timestamp-pattern', "
```